source/tests/yats-test/constraints_test.cpp

```diff
@@ -0,0 +1,34 @@
+#include <gmock/gmock.h>
+
+#include <yats/constraint.h>
+
+using namespace yats;
+
+TEST(constraints_test, remove_any_thread)
+{
+    auto constraint = thread_group::any_thread();
+
+    {
+        bool contains_any = constraint.names().count(thread_group::any_thread_name()) == 1;
```

Comment:
change this to `EXPECT_EQ` so we get better errors on failures

Reply:
EXPECT_EQ actually produces worse output (at least in my opinion)
EXPECT_FALSE: ...error: Value of: contains_any
Actual: true
Expected: false
EXPECT_EQ: ... error: Expected equality of these values:
  contains_any
    Which is: true
  false

Reply:
I meant: 
`EXPECT_EQ(constraint.names().count(thread_group::any_thread_name()), 1);`

this way, we know if there are 0 or >1 values